packages/server/src/api/routes/datasource.ts

```diff
@@ -66,5 +66,10 @@ router
     authorized(permissions.BUILDER),
     datasourceController.destroy
   )
+  .get(
+    "/api/datasources/:datasourceId/external-schema",
```

Comment:
not a fan of a dash in the URL - I think in this case it would be fine if it were just `schema`, I don't think we need to mark it as external as the whole datasources API is external (the internal tables never use the datasources API).

Reply:
This can be confused by the internal schema. In fact we have a `POST /schema` that sets the internal schema. Having a `POST` that sets the internal a schema and the same `GET` that gets the external schema can be confusing.
I don't really like the hyphen either, but I think it's what we use in other urls.
Another alternative is to have `/api/datasources/:datasourceId/schema/external`. Thoughts?

Reply:
Sure - `schema/external` works as well!